### ISSUE TITLE
Pip install fails on Python 3.9 because of lazperf

### ISSUE THREAD

**Issue by egpbos** [eEcoLiDAR/Laserfarm#50]
`pip install` from source and pypi fail on the `lazperf` dependency which only is released for Python 3.6, 3.7 and 3.8. We tried this on macOS in a `conda` environment (also tried with `mamba`). Here is the output of `pip install`:

<details>
  <summary>Click to expand!</summary>

```
Processing /Users/pbos/code/Laserfarm
  Preparing metadata (setup.py) ... done
Collecting asyncssh
  Downloading asyncssh-2.11.0-py3-none-any.whl (343 kB)
     ━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━ 343.1/343.1 kB 5.1 MB/s eta 0:00:00
Collecting dask[complete]
  Downloading dask-2022.6.0-py3-none-any.whl (1.1 MB)
     ━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━ 1.1/1.1 MB 13.1 MB/s eta 0:00:00
Collecting fire
  Downloading fire-0.4.0.tar.gz (87 kB)
     ━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━ 87.7/87.7 kB 3.1 MB/s eta 0:00:00
  Preparing metadata (setup.py) ... done
Collecting lazperf
  Downloading lazperf-1.5.tar.gz (6.0 kB)
  Preparing metadata (setup.py) ... error
  error: subprocess-exited-with-error

  × python setup.py egg_info did not run successfully.
  │ exit code: 1
  ╰─> [8 lines of output]
      /private/var/folders/rg/nyd_2nh11f50dkq0yxhdl7pc0000gn/T/pip-install-5320x1zt/lazperf_5218c732be1743dfba7be47b27096c4b/setup.py:53: DeprecationWarning: distutils Version classes are deprecated. Use packaging.version instead.
        module_version = StrictVersion(line.split("=")[1].strip().strip("\"'"))
      Traceback (most recent call last):
        File "<string>", line 2, in <module>
        File "<pip-setuptools-caller>", line 34, in <module>
        File "/private/var/folders/rg/nyd_2nh11f50dkq0yxhdl7pc0000gn/T/pip-install-5320x1zt/lazperf_5218c732be1743dfba7be47b27096c4b/setup.py", line 67, in <module>
          with open("python/README.rst", "r", **open_kwds) as fp:
      FileNotFoundError: [Errno 2] No such file or directory: 'python/README.rst'
      [end of output]

  note: This error originates from a subprocess, and is likely not a problem with pip.
error: metadata-generation-failed

× Encountered error while generating package metadata.
╰─> See above for output.

note: This is an issue with the package mentioned above, not pip.
hint: See above for details.
```
</details>

Even on Python 3.8, with pip install from PyPI we get the same error. Installing from source on Python 3.8 does work.

If this cannot be fixed or circumvented, perhaps the repo page could mention the Python version requirements.

**Comment by meiertgrootes** [comment]
@fnattino the issue raised seems to be related to lazperf's python bindings, rather than to the underlying library. Pdal and the pdal-python bindings seem to build without issues. Apart from pdal the only use of lazperf was by pylas as it's compression backend. But with the migration to laspy (see #48) there shoudl be no requirement for the lazperf python bindings anymore. Do you agree? Can we simply drop lazperf after dealing with PR #52?

**Comment by meiertgrootes** [comment]
ok @fnattino. In that case I'll remove  lazperf from the requirements in PR #52. You can check it out and either simply merge, or let me know and I'll merge.

**Comment by fnattino** [comment]
Closed by #52
